recon/src/main/java/org/hps/recon/vertexing/MultipleEventsVertexingDriver.java

```diff
@@ -105,12 +105,13 @@ protected void process(EventHeader event) {
                 //accumulatedTracks.add(track);
                 accumulatedBTracks.add(new BilliorTrack(track));
                 
-                if (track.getTrackStates().get(0).getTanLambda() > 0) {
+                //Avoid tracks too close to the beamline to have a more reliable result on z from single volumes
+                if (track.getTrackStates().get(0).getTanLambda() > 0.035) {
```

Comment:
In general, I think these configuration parameters should be put into a steering file and not hard coded. Hard coding selections should be avoided in general.

Reply:
Yeah. But this code is only for monitoring.